src/extras/helpers/PolarGridHelper.js

```diff
@@ -0,0 +1,94 @@
+import { LineSegments } from '../../objects/LineSegments';
+import { VertexColors } from '../../constants';
+import { LineBasicMaterial } from '../../materials/LineBasicMaterial';
+import { Float32Attribute } from '../../core/BufferAttribute';
+import { BufferGeometry } from '../../core/BufferGeometry';
+import { Color } from '../../math/Color';
+import { _Math as Math } from '../../math/Math';
```

Comment:
I just looked at some other source and see that it's not there but Math... works anyway. Not sure where I picked that up from but I'll remove it. 👍